invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -472,7 +512,7 @@ export class RDMDepositForm extends Component {
                   ui={this.accordionStyle}
                 >
                   <RelatedWorksField
-                    options={this.vocabularies.metadata.related_identifiers}
+                    options={this.vocabularies.metadata.identifiers}
```

Comment:
moderate: I think think related works keep going to the `related_identifiers` field (https://inveniordm.docs.cern.ch/reference/metadata/#related-identifiers-0-n).

Reply:
If the user wants to change/customize it, needs to create a component. So in any case needs to touch code (there is no easy config way for him/her to do so). Then, I think is okay to do it as you did :)